docs/api/v1.24.md

```diff
@@ -486,6 +486,8 @@ Create a container
             The default is not to restart. (optional)
             An ever increasing delay (double the previous delay, starting at 100mS)
             is added before each restart to prevent flooding the server.
+            If exposed container ports are not mapped to specific host ports
+            the restart policy may reallocate new host ports for them.
```

Comment:
This is not really a behaviour of the _restart policy_, but of the way ports are allocated. I think we should document this on the `PublishAllPorts` property instead. That currently has the following description:

> **PublishAllPorts** - Allocates a random host port for all of a container's exposed ports. Specified as a boolean value.

Thinking about adding some information there, for example:

```markdown
**PublishAllPorts** - Allocates an ephemeral host port for all of a container's
exposed ports. Specified as a boolean value.

Ports are de-allocated when the container stops, and allocated when the container
starts. The allocated port can change when restarting the container.

On Linux the port is selected from the ephemeral port range defined by
`/proc/sys/net/ipv4/ip_local_port_range`.
```